account/views.py

```
@@ -17,6 +17,12 @@
 )
 from .models import UserProfile
 
+from .serializers import (
+    UserSerializer,
```

Ruff (0.11.9)

Redefinition of unused `UserSerializer` from line 15

Remove definition: `UserSerializer`

(F811)